src/server/rpc/procedures/chart/chart.js

```diff
@@ -0,0 +1,105 @@
+const ApiConsumer = require('../utils/api-consumer'),
+    ChartNode = require('chartjs-node');
+
+let chart = new ApiConsumer('chart');
```

Comment:
why are you subclassing `ApiConsumer`? It doesn't look like you are using any external api....